src/Console/Commands/Model/Apply/ApplyCommand.cs

```diff
@@ -320,6 +351,15 @@ private async Task ApplyWebComponentChanges(string name, WebComponent entity)
                 Console.WriteLine($"Failed to apply WebComponent {name}.");
         }
 
+        private async Task ApplyThemeChanges(string name, Theme entity)
+        {
+            var applySuccessfully = await _themeApplyService.ReplaceData(Tenant, Environment,
+                            ExtractEntityNameFromFileName(name, string.Empty), entity).ConfigureAwait(false);
```

Comment:
I think we don't need to use the method ExtractEntityNameFromFileName here. Also, review the WebComponents.